apps/src/studio/studio.js

```diff
@@ -2207,40 +2308,84 @@ function cellId(prefix, row, col) {
   return prefix + '_' + row + '_' + col;
 }
 
-Studio.drawWallTile = function (svg, row, col) {
-
-  // Placeholder implementation: just drawing boxes with X's in them for now:
+/**
+ * Draw a debug rectangle centered on the given location, using the given
+ * CSS class name.
+ */
 
-  var backgroundId = cellId('wallBackground', row, col);
-  var textId = cellId('wallLetter', row, col);
+Studio.drawDebugRect = function(className, x, y, width, height) {
+  if (!showDebugInfo) {
+    return;
+  }
 
+  var svg = document.getElementById('svgStudio');
   var group = document.createElementNS(SVG_NS, 'g');
+  group.setAttribute('class', className);
   var background = document.createElementNS(SVG_NS, 'rect');
-  background.setAttribute('id', backgroundId);
-  background.setAttribute('width', Studio.SQUARE_SIZE);
-  background.setAttribute('height', Studio.SQUARE_SIZE);
-  background.setAttribute('x', col * Studio.SQUARE_SIZE);
-  background.setAttribute('y', row * Studio.SQUARE_SIZE);
+  background.setAttribute('width', width);
+  background.setAttribute('height', height);
+  background.setAttribute('x', x - width/2);
+  background.setAttribute('y', y - height/2);
   background.setAttribute('fill', 'rgba(255, 255, 255, 0.5)');
   background.setAttribute('stroke', '#000000');
   background.setAttribute('stroke-width', 1);
   group.appendChild(background);
-
-  var text = document.createElementNS(SVG_NS, 'text');
-  text.setAttribute('id', textId);
-  text.setAttribute('class', 'wall-letter');
-  text.setAttribute('width', Studio.SQUARE_SIZE);
-  text.setAttribute('height', Studio.SQUARE_SIZE);
-  text.setAttribute('x', (col + 0.5) * Studio.SQUARE_SIZE);
-  text.setAttribute('y', (row + 1) * Studio.SQUARE_SIZE - 12);
-  text.setAttribute('font-size', 32);
-  text.setAttribute('text-anchor', 'middle');
-  text.setAttribute('font-family', 'Verdana');
-  text.textContent = 'X';
-  group.appendChild(text);
   svg.appendChild(group);
 };
 
+/**
+ * Clear the debug rectangles.
+ */
+
+Studio.clearDebugRects = function() {
+  $(".spriteCenter").remove();
+  $(".itemLocation").remove();
+  $(".itemBottom").remove();
+  $(".spriteBottom").remove();
+  $(".avatarCollision").remove();
+  $(".wallCollision").remove();
+  $(".itemCollision").remove();
+  $(".spriteCollision").remove();
+  $(".chaseFreeCollision").remove();
+  $(".spriteForDistance").remove();
+  $(".itemForDistance").remove();
+  $(".spriteForChaseFree").remove();
+  $(".itemForChaseFree").remove();
+  $(".roamGridDest").remove();
+  $(".itemCenter").remove();
+  $(".roamGridPossibleDest").remove();
+};
```

Comment:
You could give each of your debug rects a unique class and a shared class (i.e. spriteCenter debugRect) which would then allow you to just do $(".debugRect").remove().
Is there a reason that each of these need unique classNames?